runners/spark/job-server/build.gradle

```diff
@@ -49,14 +49,12 @@ configurations.all {
 }
 
 dependencies {
-  compile project(sparkRunnerProject)
   compile project(path: sparkRunnerProject, configuration: "provided")
   validatesPortableRunner project(path: sparkRunnerProject, configuration: "testRuntime")
   validatesPortableRunner project(path: sparkRunnerProject, configuration: "provided")
   validatesPortableRunner project(path: ":sdks:java:core", configuration: "shadowTest")
   validatesPortableRunner project(path: ":runners:core-java", configuration: "testRuntime")
   validatesPortableRunner project(path: ":runners:portability:java", configuration: "testRuntime")
-  compile project(":sdks:java:extensions:google-cloud-platform-core")
```

Comment:
I believe we need this as a runtime-only dependency for the expansion service.

Reply:
corrected,thanks!